client/reader/search-stream/empty.jsx

```diff
@@ -41,7 +41,7 @@ class SearchEmptyContent extends React.Component {
 				onClick={ this.recordAction }
 				href="/read"
 			>
-				{ this.props.translate( 'Back to Following' ) }
+				{ this.props.translate( 'Back to following' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.